kubernetes/server/create.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
 function kubectl_create() {
-  kubectl create -f scdf.yml
+  kubectl create -f scdf.yml --namespace $KUBERNETES_NAMESPACE
   READY_FOR_TESTS=1
   for i in $( seq 1 "${RETRIES}" ); do
-    SERVER_URI=$(kubectl get svc scdf | grep scdf | awk '{print $3}')
+    SERVER_URI=$(kubectl get svc scdf --namespace $KUBERNETES_NAMESPACE | grep scdf | awk '{print $3}')
     [ '<pending>' != $SERVER_URI ] && READY_FOR_TESTS=0 && break
```

Comment:
Do you have an older `kubectl` version, the newer ones say <pending> when the external IP hasn't been assigned yet